### ISSUE TITLE
NameplateCooldowns config.

### ISSUE THREAD

**Issue by FelipeOrtuzarSawred** [casualshammy/NameplateCooldowns#15]
How i can change number color to white and how i can add border to icons? As before versions

Ty

**Comment by casualshammy** [comment]
Hello.
You cannot do this, because it's never been implemented.

**Comment by FelipeOrtuzarSawred** [comment]
True, u have the reason respect to number color, but the borders, there is a image show icons with border and is a before versions. If u can implement this option or visual graphic, I would be very grateful.

Ty 

![Example](https://user-images.githubusercontent.com/64238286/85933633-39621580-b8a7-11ea-9241-994129f9a675.jpg)

**Comment by casualshammy** [comment]
So this feature isn't working now? Could you please post a screenshot with 'General' settings of NameplateCooldowns?

**Comment by FelipeOrtuzarSawred** [comment]
Of course, this image is shot with test mode activated, but is same visualization when play with addon activate in fight versus enemy player.

![WoWScrnShot_062820_060103](https://user-images.githubusercontent.com/64238286/85944478-08183280-b905-11ea-91d0-71c152a5a36b.jpg)

**Comment by casualshammy** [comment]
Sorry, seems like I've misunderstood you. You WANT borders around **interruptions** and **trinkets** to be shown, but this feature is NOT working, right?

**Comment by FelipeOrtuzarSawred** [comment]
Maybe I expressed myself wrong, the edges of TRINKETS and INTERRUPTIONS work well, but what I meant is the edges of the rest of the skills are not there, if you look closely the skills look straight and not rounded as in the first picture.

**Comment by casualshammy** [comment]
Ah I see the difference. The first screenshot was made a looooong ago. In those days I did not know how to remove this standard rounded borders around icons. If you want, I can make an option to return this borders.

**Comment by FelipeOrtuzarSawred** [comment]
For me it would be excellent and I would appreciate it very much. It is just some aesthetics, for all addons they look the same.

**Comment by casualshammy** [comment]
Implemented in the latest release (80300.2-release).

**Comment by FelipeOrtuzarSawred** [comment]
Thank you my friend, I am very grateful.

How can I help you now?

**Comment by casualshammy** [comment]
I will be glad if you continue using my addon ;)22:19, 2020年6月29日, FelipeOrtuzarSawred <notifications@github.com>:
Thank you my friend, I am very grateful.
How can I help you now?

—You are receiving this because you were assigned.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Sent from Yandex.Mail for mobile
